api/verify.go

```diff
@@ -331,9 +336,10 @@ func (a *API) emailChangeVerify(ctx context.Context, conn *storage.Connection, p
 		if err != nil {
 			return nil, err
 		}
-		return nil, acceptedTokenError("Email change request accepted")
+		return nil, acceptedTokenError("Email change request accepted").WithInternalError(redirectWithQueryError)
```

Comment:
missed out on redirecting a user if only 1 link is confirmed, this is the fix that is needed